src/mlpack/tests/feedforward_network_test.cpp

```diff
@@ -246,10 +249,10 @@ TEST_CASE("ForwardBackwardTest", "[FeedForwardNetworkTest]")
   ens::VanillaUpdate opt;
   model.ResetParameters();
   #if ENS_VERSION_MAJOR == 1
-  opt.Initialize(model.Parameters().n_rows, model.Parameters().n_cols);
+    opt.Initialize(model.Parameters().n_rows, model.Parameters().n_cols);
```

Comment:
```suggestion
  opt.Initialize(model.Parameters().n_rows, model.Parameters().n_cols);
```

Indentation issue here.